### ISSUE TITLE
Map grey out

### ISSUE THREAD

**Issue by nguye3hg** [JustFly1984/react-google-maps-api#3244]
# Issue template

You can donate or became a sponsor [https://opencollective.com/react-google-maps-api#category-CONTRIBUTE](https://opencollective.com/react-google-maps-api#category-CONTRIBUTE)

If you want to ask question, please ask it in Github Discussions, [Spectrum.chat](https://spectrum.chat/react-google-maps) or [Slack channel](https://join.slack.com/t/react-google-maps-api/shared_invite/enQtODc5ODU1NTY5MzQ4LTBiNTYzZmY1YmVjYzJhZThkMGU0YzUwZjJkNGJmYjk4YjQyYjZhMDk2YThlZGEzNDc0M2RhNjBmMWE4ZTJiMjQ)

Please do not post unformatted code into issues, and please do not ask questions. Only real issues, PR's or feature requests are allowed. Minimal reproduction in codesandbox.io is required.

## Please provide an explanation of the issue
My Map is Grey Out like this when I go to any location Outside of the US. I can not put markers outside of the US to make them appear on the Map as well

![Screenshot (393)](https://github.com/JustFly1984/react-google-maps-api/assets/88401567/68a73e28-14b6-46a5-84a1-4bd99e0b40ca)

## Your Environment

os: mac/linux/windows/android/ios

node --version

react version

webpack version

@babel version

@react-google-maps/api version

**Comment by msidlo** [comment]
Hi @nguye3hg. I have the same issue on localhost in development since **3.54** map version. Using **3.53** may solve your issue. [V3 versions](https://developers.google.com/maps/documentation/javascript/versions).

E.g.:
```
useJsApiLoader({ googleMapsApiKey: 'API_KEY', version: '3.53' })
```

TL;DR
I've noticed the same behaviour on one of my projects. In our production everything works fine, but on localhost I see the same issue. This part of the code base wasn't changed for considerable time. In our project, the version of this library is fixed. Thus I believe the issue itself **is not caused by this library**. I'd watch for the map versions. Currently the latest is 3.54. Let's hope for now in 3.55 everything will get back to normal.

**Comment by msidlo** [comment]
I reviewed the issue today and found the problem. We use [miragejs](https://github.com/miragejs/miragejs/issues/1092) for API mocking and it is interfering with google maps. It's interesting it was working until 3.54 version without any problem.

For the reference if anybody encounter this issue, adding following passthrough into miragejs server solves the problem.
```TypeScript
this.passthrough('https://maps.googleapis.com/**')
```
